eq-author/src/App/page/Design/Validation/GroupValidations/TotalValidation/TotalValidationEditor/index.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { flowRight } from "lodash";
+import styled from "styled-components";
+import { propType } from "graphql-anywhere";
+import PropTypes from "prop-types";
+
+import { Select } from "components/Forms";
+import { Grid, Column } from "components/Grid";
+import withEntityEditor from "components/withEntityEditor";
+
+import withChangeUpdate from "enhancers/withChangeUpdate";
+import withPropRemapped from "enhancers/withPropRemapped";
+
+import totalFragment from "graphql/fragments/total-validation-rule.graphql";
+
+import { ValidationPills } from "../../../ValidationPills";
+import ValidationTitle from "../../../ValidationTitle";
+import PathEnd from "../../../path-end.svg?inline";
+import withUpdateValidationRule from "../../../withUpdateValidationRule";
+import { numericReadToWriteMapper } from "../../../readToWriteMapper";
+
+import CustomEditor from "./CustomEditor";
+import PreviousAnswerEditor from "./PreviousAnswerEditor";
+
+const Connector = styled(PathEnd)`
+  display: block;
+  margin-left: auto;
+`;
+
+const Flex = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const EqualsSelect = styled(Select)`
+  width: 8em;
+`;
+
+const Pills = styled(ValidationPills)`
+  margin-top: -2em;
+`;
+
+const Margin = styled.div`
+  margin: 0 3em;
+`;
+
+export const TotalValidationEditor = ({
+  total,
+  type,
+  onChange,
+  onUpdate,
+  onChangeUpdate,
+}) => {
+  return (
+    <Margin>
+      <Grid>
+        <Column cols={5}>
+          <Flex>
+            <ValidationTitle style={{ margin: "0 1em 0 0" }}>
+              Total
+            </ValidationTitle>
+
+            <EqualsSelect
```

Comment:
This needs a label (albeit visually hidden)

Reply:
Done